Formula/git-bug.rb

```diff
@@ -0,0 +1,38 @@
+class GitBug < Formula
+  desc "Distributed, offline-first bug tracker embedded in git, with bridges"
+  homepage "https://github.com/MichaelMure/git-bug"
+  url "https://github.com/MichaelMure/git-bug/archive/0.7.1.tar.gz"
+  sha256 "78a6c7dee159cdad4ad69066d6d8fc4b7c259d5ea6351aaf6709b6ac03ff3d2f"
+
+  depends_on "go" => :build
+
+  def install
+    system "make", "build"
+    bin.install "git-bug"
+  end
+
+  test do
+    # Version
+    assert_includes shell_output("#{bin}/git-bug --version"), "git-bug version"
+    # Version through git
+    assert_includes shell_output("git bug --version"), "git-bug version"
+    # List issues outside a repository
+    assert_includes shell_output("git bug ls 2>&1", 1), "Error: git-bug must be run from within a git repo"
+    # Add issue outside a repository
+    assert_includes shell_output("git bug add 2>&1", 1), "Error: git-bug must be run from within a git repo"
+    # Push issues to a remote from outside a repository
+    assert_includes shell_output("git bug push origin 2>&1", 1), "Error: git-bug must be run from within a git repo"
+    # Pull issues from a remote from outside a repository
+    assert_includes shell_output("git bug pull origin 2>&1", 1), "Error: git-bug must be run from within a git repo"
+    
+    mkdir testpath/"git-repo" do
+      output = shell_output("git init")
+      output = shell_output("git config user.name homebrew;git config user.email a@a.com")
+      output = shell_output("yes \"a b http://www/www\" | git bug user create")
+      output = shell_output("git bug add -t \"Issue 1\" -m \"Issue body\"")
+      output = shell_output("git bug add -t \"Issue 2\" -m \"Issue body\"")
+      output = shell_output("git bug add -t \"Issue 3\" -m \"Issue body\"")
```

Comment:
Don't assign `output` if it's not used.
Use `system` instead of `shell_output`

Reply:
I tried to use system initially but couldn't find how to specify the pipe of the third command, sorry, I haven't used Ruby in the last decade :)

Reply:
That's alright, here is an example https://github.com/Homebrew/homebrew-core/blob/4d24c3984a390bc931c1b9c1fd0f6ed724c49e7b/Formula/zola.rb#L26

Reply:
Done.